scrapers/JavLibrary_python.py

```diff
@@ -269,7 +269,14 @@ def r18_search(html, xpath):
         r18_result["series_name"] = r18_search_serie
     if r18_search_url:
         r18_search_url = r18_search_url[0]
-        r18_main_html = sendRequest(r18_search_url, R18_HEADERS)
+        r18_id = re.match(r".+id=(.+)\/.+", r18_search_url)
```

Comment:
should be
```python
        r18_id = re.match(r".+id=(.+)/.*", scene_url)
```

as `/` doesnt need to be escaped
and urls may be clean already so nothing after the `id=xxxx/` part